plugins/vega/web_client/views/VegaWidget.js

```diff
@@ -29,13 +30,13 @@ var VegaWidget = View.extend({
             restRequest({
                 url: `item/${this.item.id}/download`
             })
-                .done(function (spec) {
+                .done(_.bind(function (spec) {
                     let runtime = parse(spec);
                     let view = new VegaView(runtime)
-                        .initialize(document.querySelector('.g-item-vega-vis'))
+                        .initialize($('.g-item-vega-vis')[0])
```

Comment:
Can this be `this.$`? It's nice if we can scope these queries, but I realize sometimes the element to be selected is not a direct child of the current view's `this.el`.

Reply:
I believe the latter is the case here.